routers/mainRouter.js

```diff
@@ -0,0 +1,9 @@
+const { Router } = require('express')
```

Comment:
I'd like you to make it a bit easier:
```suggestion
const router = require('express').Router();
```

Reply:
ok